bedrock/base/templates/macros.html

```diff
@@ -412,7 +412,7 @@ <h2 class="fxa-email-form-title">{{ form_title }}</h2>
         <input type="email" name="email" id="fxa-email-field" class="fxa-email-field" placeholder="user@example.com" required>
       </p>
 
-      <button type="submit" class="{{ button_class }}" id="fxa-email-form-submit" data-link-type="FxA-{{ service|title }}" data-link-name="Continue">
+      <button type="submit" class="js-fxa-cta-link {{ button_class }}" id="fxa-email-form-submit" data-link-type="FxA-{{ service|title }}" data-link-name="Continue">
```

Comment:
Placing this class on a `<button>` results in the error `TypeError: oldAccountsLink is undefined`, since the JS code expects an `href` property. (this can be observed when loading /accounts with a valid query param).

To handle attribution on form submissions, I imagine we might need a separate piece of logic to append / update hidden inputs with utm values. Thoughts?

Reply:
Yeah, I think the forms need to be addressed in a separate PR. I've removed the class here to avoid the error.

Reply:
✅